sympy/assumptions/sathandlers.py

```diff
@@ -114,21 +106,13 @@ class AnyArgs(UnevaluatedOnFree):
 
     The typical usage is to evaluate predicates with expressions using .rcall().
 
-    Example
-    =======
-
-    >>> from sympy.assumptions.sathandlers import AnyArgs
-    >>> from sympy import symbols, Q
-    >>> x, y = symbols('x y')
-    >>> a = AnyArgs(Q.positive & Q.negative)
-    >>> a
-    AnyArgs(Q.negative & Q.positive)
-    >>> a.rcall(x*y)
-    (Q.negative(x) & Q.positive(x)) | (Q.negative(y) & Q.positive(y))
     """
+    # TODO: Add examples.
```

Comment:
Why have you removed the examples from the docstrings?

Reply:
```py
>>> from sympy.abc import x,y,z
>>> from sympy.assumptions.sathandlers import *
>>> AnyArgs(Q.positive(x*y))
(Literal(Q.positive(x), False) | Literal(Q.positive(y), False))
>>> 
```
The current results of them aren't expressive enough to be put as examples and they may change as well. We can add examples once everything becomes stable.

Reply:
I think we should keep the examples. The code is for internal use only, so the examples are only there to help people reading the code to understand how it works.